int_experian/int_experian_sfra/cartridge/static/default/js/edqUtils.js

```diff
@@ -502,12 +500,10 @@ function setCheckoutFormEvents() {
 }
 function setEventsForListeners(checkoutStage) {
 	setEdqInputSelectors(checkoutStage);
-	removeMultipleEDQSuggestion();
 	edqSetGlobalIntuitiveConfiguration();
 	addEventOnElement("[name=" + edqAddressLine1Id.name + "]", "keypress", setCountryField);
 	addEventOnElement("[name=" + edqAddressLine1Id.name + "]", "mouseleave", setCountryField);
 	addEventOnElement("[name=" + edqAddressLine1Id.name + "]", "mouseenter", setCountryField);
-	EDQ.address.globalIntuitive.activateValidation(edqAddressLine1Id);
```

Comment:
Removed this line since the usage of this function was creating more Edq Global Intuitive - Address-Suggestion boxes and the index where misbehaving.